Gemfile

```diff
@@ -37,6 +37,9 @@ gem 'countries'
 #select country info in the ISO standard
 gem 'country_select'
 
+#parse html
+gem 'nokogiri', '~>1.6.8'
```

Comment:
When I remove the gem's version numbers from the Gemfile and run `bundle install`, it also removes the gem's version numbers from the Gemfile.lock. How can I know the versions I used later on?